lib/taskana-core-test/src/test/java/acceptance/taskcomment/create/CreateTaskCommentAccTest.java

```diff
@@ -120,11 +123,17 @@ void should_CreateTaskComment_When_CopyingAnotherComment() throws Exception {
         taskService.getTaskComments(taskWithComments.getId());
     assertThat(taskCommentsAfterInsert)
         .containsExactlyInAnyOrder(comment1, comment2, taskCommentToCreate);
+    assertThat(taskService.getTask(taskWithComments.getId()).getNumberOfComments()).isEqualTo(3);
+
+    // Deleting the comment so that the comments remain the same in different tests inside this
+    // class
+    taskService.deleteTaskComment(taskCommentToCreate.getId());
   }
 
   @WithAccessId(user = "user-1-1")
   @Test
-  void should_FailToCreateTaskComment_When_TaskIdIsNullOrNonExisting() throws Exception {
+  void should_FailToCreateTaskCommentAndUpdateNumberOfComments_When_TaskIdIsNullOrNonExisting()
```

Comment:
What do you think about should_FailToCreateTaskCommentAndNotUpdateNumberOfComments_When_TaskIdIsNullOrNonExisting()
instead?

Reply:
yes that would be a better name, i will rename it.